src/website/src/styles/home.scss

```diff
@@ -104,9 +104,17 @@
     position: absolute;
     bottom: 0;
     left: 612px;
-    background-image: url('../assets/images/clarity_logo.svg');
+    background-image: url('../assets/images/clarity-logo.svg');
     background-size: cover;
     background-position: center center;
+
+    &--spring {
```

Comment:
Minor, but we are not trying to do BEM in our sass, though its in a few places.

Reply:
Way back a long time ago we were trying to use the [SUITECSS naming conventions](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md)

But it wasn't always uniformly applied across the code base. And, I don't think we have documented it anywhere. I learned about it through discussions w/ @adityarb88 (👋🏼- we miss you :) )